apps/omg/mix.exs

```diff
@@ -46,4 +46,19 @@ defmodule OMG.MixProject do
       {:plug_cowboy, "~> 1.0", only: [:dev, :test]}
     ]
   end
+
+  defp version_and_git_revision_hash do
+    {rev, _i} = System.cmd("git", ["rev-parse", "HEAD"])
+    sha = String.replace(rev, "\n", "") |> Kernel.binary_part(0, 7)
```

Comment:
`git rev-parse --short=7 HEAD` can help make this smaller maybe?

Reply:
yeah, but not by much :)
string.replace/3 is still needed.